elections/2021-TOC2/election.yaml

```diff
@@ -0,0 +1,15 @@
+name: Knative 2021 TOC Special Election
+organization: Knative
+start_datetime: 2021-07-01 00:00:01
+end_datetime: 2021-07-15 23:59:00
+no_winners: 2
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - employer
+election_officers:
+  - geekygirldawn
+  - jberkus
+eligibility: All Knative contributors with more than 50 contributions are eligible to vote
```

Comment:
eligibility: all Knative contributors who were eligible to vote in the original 2021 TOC election are eligible to vote in this one

Reply:
... since we're not drawing a new list